x-pack/plugins/observability_solution/observability_ai_assistant_management/public/routes/components/knowledge_base_edit_user_instruction_flyout.tsx

```diff
@@ -118,7 +117,6 @@ export function KnowledgeBaseEditUserInstructionFlyout({ onClose }: { onClose: (
               fill
               isLoading={isSaving}
               onClick={handleSubmit}
-              isDisabled={isSubmitDisabled}
```

Comment:
Hi @sorenlouv 
I'm unable to clear the user-specific system prompt. 
It gives an error..

```
Error: Failed to validate: 
  in /body/text: "" does not match expected type NonEmptyString
    at Object.onError (http://localhost:5601/xat/XXXXXXXXXXXX/bundles/plugin/observabilityAiAssistantManagement/1.0.0/observabilityAiAssistantManagement.chunk.0.js:24903:23)
    at Mutation.execute (http://localhost:5601/xat/XXXXXXXXXXXX/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:196451:123)
    at async handleSubmit (http://localhost:5601/xat/XXXXXXXXXXXX/bundles/plugin/observabilityAiAssistantManagement/1.0.0/observabilityAiAssistantManagement.chunk.0.js:26014:5)
```


https://github.com/user-attachments/assets/e2675ffe-6b03-40df-9421-1370d84cfc42

Reply:
Thanks for catching that @viduni94! That should be fixed now along with an API test